juniper_axum/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "juniper_axum"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.62"
+description = "`juniper` GraphQL integration with `axum`."
+license = "BSD-2-Clause"
+authors = ["Benno Tielen <Benno@tielen.nl>"]
+documentation = "https://docs.rs/juniper_axum"
+homepage = "https://github.com/graphql-rust/juniper/tree/master/juniper_axum"
+repository = "https://github.com/graphql-rust/juniper"
+readme = "README.md"
+categories = ["asynchronous", "web-programming", "web-programming::http-server"]
+keywords = ["graphql", "juniper", "axum", "websocket"]
+exclude = ["/release.toml"]
+
+[dependencies]
+axum = { version = "0.5.11", features = ["ws"]}
+juniper = { path = "../juniper" }
```

Comment:
As mentioned in the PR comments already, this should have a version and wired up to the release machinery.